src/components/Authentication/Login/index.js

```diff
@@ -1,30 +1,96 @@
 import React, { Component } from 'react';
 import {
-  StyleSheet, Text, View,
+  Text, View, TouchableOpacity, Image, ActivityIndicator,
 } from 'react-native';
-import { TouchableOpacity } from 'react-native-gesture-handler';
-
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-    backgroundColor: '#F5FCFF',
-  },
-});
+import { connect } from 'react-redux';
+import { changeToSignupComponent } from 'datalayer/actions/auth.action';
+import logoPath from 'assets/images/Easy-Event.png';
+import InputField from 'components/Authentication/InputField';
+import styles from './index.styles';
 
 class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      loading: false,
+    };
+  }
+
+  // If I remove this method it will make error of "Unused text fields" on the inputForm
+  getInitialState = () => {
+    const { email, password } = this.state;
+    return {
+      email,
+      password,
+    };
+  }
+
+  onLogin = () => {
+
+  }
+
+  renderLoginButton = () => {
+    const { loading } = this.state;
+    if (!loading) {
+      return (
+        <View>
+          <TouchableOpacity
+            style={styles.button}
+            onPress={this.onLogin}
+          >
+            <Text style={styles.text}>LOGIN</Text>
+          </TouchableOpacity>
+        </View>
+      );
+    }
+    return (
+      <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+        <ActivityIndicator size="large" color="#fb3" />
+      </View>
+    );
+  }
+
   render() {
-    const { navigation } = this.props;
+    const { changeToSignupComponent } = this.props;
     return (
       <View style={styles.container}>
-        <Text>This is a login component!</Text>
-        <TouchableOpacity onPress={() => navigation.navigate('App')}>
-          <Text>Go to Home Page</Text>
-        </TouchableOpacity>
+        <View style={styles.imgContainer}>
+          <Image
+            style={styles.imgLogo}
+            source={logoPath}
+          />
+        </View>
+
+        <View style={styles.formContainer}>
+          <InputField
+            placeholder="Email..."
+            setText={(text) => this.setState({ email: text })}
```

Comment:
Write a method: `onChangeText`
Reference: https://stackoverflow.com/questions/44416541/react-native-difference-between-onchange-vs-onchangetext-of-textinput